src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,7 @@
+
+#cloud config for gateway
+spring:
  cloud:
    config:
      uri:
      - http://localhost:8086/

```

Comment:
Same as above.

Reply:
Fixed. Was the line delimiters configuration problem.